### PR TITLE
fesom: add echam6 as forcing, echam: / added to paths

### PR DESCRIPTION
**Pull request by mandresm**
Some additional changes to get `AWICM-CMIP6` running.

Before merging, needs testing.

### CODE REVIEW

File: configs/components/fesom/fesom.yaml [hunk 1]
```diff
@@ -16,8 +16,6 @@ available_versions:
 - '2.0'
 - '2.0-r' # OG: temporarily here
 - '2.0-jio'
-- '2.1'
```
Comment (mandresm) [inline configs/components/fesom/fesom.yaml]: @vkolatschek , so you know why did you need to delete this line?

File: configs/components/fesom/fesom.yaml [hunk 2]
```diff
@@ -128,9 +131,9 @@ restart_in_in_work:
         oce_restart: fesom.${parent_date!syear}.oce.nc
         ice_restart: fesom.${parent_date!syear}.ice.nc
 restart_in_sources:
-        oce_diag: fesom.${parent_date!syear}.oce.diag.nc
-        oce_restart: fesom.${parent_date!syear}.oce.nc
-        ice_restart: fesom.${parent_date!syear}.ice.nc
+        oce_diag: fesom.${parent_date!syear!smonth!sday}.oce.diag.nc
+        oce_restart: fesom.${parent_date!syear!smonth!sday}.oce.nc
+        ice_restart: fesom.${parent_date!syear!smonth!sday}.ice.nc
```
Comment (mandresm) [inline configs/components/fesom/fesom.yaml]: This needs to be reverted back

File: configs/components/fesom/fesom.yaml [hunk 3]
```diff
@@ -143,9 +146,9 @@ restart_out_in_work:
         ice_restart: fesom.${end_date!syear}.ice.nc
 
 restart_out_sources:
-        oce_diag: fesom.${end_date!syear}.oce.diag.nc
-        oce_restart: fesom.${end_date!syear}.oce.nc
-        ice_restart: fesom.${end_date!syear}.ice.nc
+        oce_diag: fesom.${end_date!syear!smonth!sday}.oce.diag.nc
+        oce_restart: fesom.${end_date!syear!smonth!sday}.oce.nc
+        ice_restart: fesom.${end_date!syear!smonth!sday}.ice.nc
```
Comment (mandresm) [inline configs/components/fesom/fesom.yaml]: This needs to be reverted back

File: configs/components/fesom/fesom.yaml [hunk 4]
```diff
@@ -165,8 +168,7 @@ log_sources:
 namelist_changes:
         namelist.config:
                 clockinit:
-                        daynew: "${daynew}"
-                        yearnew: "${yearnew}"
+                        yearnew: "${initial_date!syear}"
```
Comment (mandresm) [inline configs/components/fesom/fesom.yaml]: This was fixed in a different way in another commit and needs to be reverted back

File: configs/components/fesom/fesom.yaml [hunk 5]
```diff
@@ -222,18 +224,15 @@ choose_lresume:
...
-
+                choose_general.run_number:
+                        # MAM: If it is a restart triggered from a runscript
+                        # it is necessary to specify the year of the initial
+                        # date of the experiment that is to be continued
+                        1:
+                                add_namelist_changes:
+                                        namelist.config:
+                                                clockinit:
+                                                        yearnew: "${first_initial_year}"
```
Comment (mandresm) [inline configs/components/fesom/fesom.yaml]: Also needs to be reverted back

File: configs/components/fesom/fesom.yaml [hunk 6]
```diff
@@ -260,9 +259,10 @@ yearly_outputs: [lwrd,olat,olwout,osen,relax_salt,shum,tair,uwind,vwind]
 
 monthly_outputs: [evs,fsitherm,hfds,opottemptend,pbo,prlq,prsn,rsdo,siarean,siareas,sidmassevapsubl,sidmasssi,sidmassth,sidmasstranx,sidmasstrany,siextentn,siextents,sifllatstop,sisnconc,sisnmass,sisnthick,sispeed,sivol,sivoln,sivols,soga,thetaoga,u2o,uo,uso,uto,v2o,vo,volo,vso,vto,w2o,wfo,wo,wso,wto,zossq,rho,uhice,uhsnow,urho,uv,vhice,vhsnow,virtual_salt,vrho]
 
-daily_outputs: [mlotst,omldamax,sic,sistrxdtop,sistrxubot,sistrydtop,sistryubot,sithick,sitimefrac,siu,siv,so,sos,tauuo,tauvo,thetao,tos,zos,flice,wnet,evap,runoff,thdgr,thdgrsn]
+# daily_outputs: [mlotst,omldamax,sic,sistrxdtop,sistrxubot,sistrydtop,sistryubot,sithick,sitimefrac,siu,siv,so,sos,tauuo,tauvo,thetao,tos,zos,flice,wnet,evap,runoff,thdgr,thdgrsn]
+daily_outputs: []
```
Comment (mandresm) [inline configs/components/fesom/fesom.yaml]: This should go into the runscript

File: configs/components/fesom/fesom.yaml [hunk 7]
```diff
@@ -260,9 +259,10 @@ yearly_outputs: [lwrd,olat,olwout,osen,relax_salt,shum,tair,uwind,vwind]
...
 
-12_step_outputs: [tso]
+24_second24_secondts: [tso]
```
Comment (mandresm) [inline configs/components/fesom/fesom.yaml]: I don't think this variable does anything, so it will be reverted back.

File: configs/components/fesom/fesom.yaml [hunk 8]
```diff
@@ -330,6 +330,10 @@ runtime_environment_changes:
                                         vars: ${12_step_outputs}
                                         unit: s
                                         rate: 12
+                                -
+                                        vars: ${1_step_outputs}
+                                        unit: s
+                                        rate: 1
```
Comment (mandresm) [inline configs/components/fesom/fesom.yaml]: This variable is not defined anywhere and will be reverted back.

File: configs/components/fesom/fesom.yaml (hunk 1 above)
Comment (vkolatschek) [reply configs/components/fesom/fesom.yaml]: I don't think I did? I did only change the naming of the restart files. Any other differences must be there, because 'my' fesom.yaml file was not up to date.

File: configs/components/fesom/fesom.yaml (hunk 8 above)
Comment (vkolatschek) [reply configs/components/fesom/fesom.yaml]: I added this, because I needed output after each time step to figure out where my model went crazy.

File: configs/components/fesom/fesom.yaml (hunk 1 above)
Comment (mandresm) [reply configs/components/fesom/fesom.yaml]: great, thanks for that info :)

**Comment by mandresm** [comment]
This branch won't be merged in the end to `monorepo`, but instead `monorepo_esm_tests_and_CMIP6` (fcf6ce82aa51a0eac154e80b8132b183556f9729).

The following tests were sucessfull

- awicm:
    - 2.0:
        - awicm2-initial-monthly:
            - mistral:  compiles        runs
            - ollie:    compiles        runs
    - CMIP6:
        - awicm1-CMIP6-initial-monthly:
            - mistral:  compiles        runs
            - ollie:    compiles        runs
- awiesm:
    - 2.1-recom:
        - all_awiesm-2.1-recom:
            - mistral:  compiles        runs
            - ollie:    compiles        runs
    - 2.1-wiso:
        - PI_ctrl_awiesm-2.1-wiso:
            - mistral:  compiles        runs
            - ollie:    compiles        runs
- echam:
    - 6.3.04p1:
        - echam6.3.04p1-initial-monthly:
            - mistral:  compiles        runs
            - ollie:    compiles        runs
- fesom:
    - 2.0:
        - fesom2.0-initial-monthly:
            - mistral:  compiles        runs
            - ollie:    compiles        runs
    - 2.1:
        - fesom2.1-initial-monthly:
            - mistral:  compiles        runs
            - ollie:    compiles        runs
- fesom-recom:
    - 1.4:
        - fesom-recom1.4-initial-daily:
            - ollie:    compiles        runs
